packages/twenty-server/src/workspace/workspace-resolver-builder/workspace-resolver.factory.ts

```diff
@@ -49,6 +51,7 @@ export class WorkspaceResolverFactory {
     >([
       ['findMany', this.findManyResolverFactory],
       ['findOne', this.findOneResolverFactory],
+      ['findDuplicates', this.findDuplicatesResolverFactory],
```

Comment:
@rostaklein Maybe we should add a condition bellow to only add query of object within the `duplicateCriteria` array.
You can by example in the `for...of` loop skip for the method name `findDuplicates` if the objectName is not matching the object metadata singular name.